### PR TITLE
Usage: show which args are optional

### PR DESCRIPTION
**Pull request by misfo**

### CODE REVIEW

File: cmd/brew-setup-nginx-conf [hunk 1]
```diff
@@ -17,7 +17,7 @@ root = ARGV.shift || "."
 input = ARGV.shift || "config/dev/nginx.conf.erb"
 
 if !name || !root || !input
-  abort "Usage: brew generate-nginx-conf [--root] <project_name> <project_root_path> <nginx.conf.erb>"
+  abort "Usage: brew generate-nginx-conf [--root] <project_name> [<project_root_path> [<nginx.conf.erb>]]"
```
Comment (MikeMcQuaid) [inline cmd/brew-setup-nginx-conf]: `<project_name> [project_root_path [nginx.conf.erb]]` would be more consistent with our typical format. Thanks though 👍

Comment (MikeMcQuaid) [inline cmd/brew-setup-nginx-conf]: @misfo ping on this?

Comment (mistydemeo) [inline cmd/brew-setup-nginx-conf]: Ping

**Comment by MikeMcQuaid** [comment]
Passing on this for now but thanks for the PR. Happy to receive a new one with comments addressed.

**Pull request closed without merging**